tests/test_dos.py

```diff
@@ -7,12 +7,32 @@
 sys.path.append('..')
 
 from qiling import Qiling
+from qiling.const import QL_INTERCEPT
+
+class Checklist:
+    def __init__(self) -> None:
+        self.visited_onenter = False
+        self.visited_onexit = False
 
 class DOSTest(unittest.TestCase):
 
     def test_dos_8086_hello(self):
-        ql = Qiling(["../examples/rootfs/8086/dos/HI.DOS_COM"], "../examples/rootfs/8086/dos")
+        ql = Qiling(["../examples/rootfs/8086/dos/HI.DOS_COM"], "../examples/rootfs/8086/dos", output='debug')
```

Comment:
i am thiniking we should remove output="" for good and keep only verbose ?

Reply:
I always found `output` and `verbose` a bit confusing, but in the current impl both are needed.
Personally, I use `output='debug'` only when I need to enable assembly-level tracing*. We should think of a way to consolidate them both into one variable that would reflect all needed verbosity levels and allow tracing.

( * see what I did for `hello_x8664_linux_disasm.py` )

Reply:
i agree. I will merge this and we need to do it next.